sympy/series/formal.py

```diff
@@ -810,6 +811,11 @@ def _compute_fps(f, x, x0, dir, hyper, order, rational, full):
             return ak, xk, ind
         return None
 
+    syms = list(f.free_symbols.difference({x}))
+    symb = S.One
+    if len(syms) == S.One:
```

Comment:
What happens for something like ``x*(m+n)*sin(x**2)``?

Reply:
I had implemented it keeping one symbol in mind. I will try to change it.

Reply:
Done. @leosartaj